### PR TITLE
added contributor cong-nguyen-dinh and blogpost

### PR DESCRIPTION
**Pull request by congnguyendinh0**
*Issue #, if available:*

*Description of changes:*
added contributor cong-nguyen-dinh and blogpost

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/contributors/cong-nguyen-dinh/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Cong NguyenßDinh
```
Comment (jamesonwilliams) [inline content/contributors/cong-nguyen-dinh/index.md:2]: This rendered as the German ß, but your name on LinkedIn is written with a hyphen, "Cong Nguyen-Dinh". Did you mean to use a `-`, here?
```suggestion
name: Cong Nguyen-Dinh 
```

**Review by jamesonwilliams — COMMENTED** [review]
Currently the `banner.png` is very text-rich. Is there a simpler image you can include?

**Comment by congnguyendinh0** [comment]
> Currently the `banner.png` is very text-rich. Is there a simpler image you can include?

@jamesonwilliams  thank you, as you suggested I have added a simpler version. Very supportive of you!

**Review by jamesonwilliams — APPROVED** [review]

**Comment by congnguyendinh0** [comment]
@dabit3  would you be so kind and review the request, thank you! ☺️

**Review by dabit3 — APPROVED** [review]

**Comment by dabit3** [comment]
Thanks @congnguyendinh0 !

**Pull request merged**